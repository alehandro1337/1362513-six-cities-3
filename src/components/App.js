import React from "react";
import Main from "./Main";

const App = (props) => {
  const {offers} = props;

  return <div>
    <h1>React Application</h1>
    <Main
      offers={offers}
    />
  </div>;
}

export default App;
