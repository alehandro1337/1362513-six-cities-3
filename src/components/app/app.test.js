import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

const names = [
  {
    title: `Name One`
  },
  {
    title: `Name Two`
  }
];

const offers = 3;

it(`<App /> should renderer 3`, () =>{
  const tree = renderer
    .create(<App
      names={names}
      offers={offers}
      onMainTitleClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
