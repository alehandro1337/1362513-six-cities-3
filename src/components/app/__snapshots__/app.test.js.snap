// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should renderer 3 1`] = `
<div>
  <h1>
    React Application
  </h1>
  <h1
    className="main-title"
    onClick={[Function]}
  >
    Количество предложений: 
    3
  </h1>
  <ul>
    <li>
      Name One
    </li>
    <li>
      Name Two
    </li>
  </ul>
</div>
`;
